rdkit/Chem/PeriodicTable.py

```diff
@@ -128,11 +127,11 @@
 for line in periodicData.split('\n'):
   splitLine = line.split()
   if len(splitLine) > 1:
-    nameTable[splitLine[1]] = (int(splitLine[0]),float(splitLine[6]),int(splitLine[7]),\
-                               int(splitLine[5]),float(splitLine[2]),float(splitLine[3]),
+    nameTable[splitLine[1]] = (int(splitLine[0]), float(splitLine[6]), int(splitLine[7]),
+                               int(splitLine[5]), float(splitLine[2]), float(splitLine[3]),
                                float(splitLine[4]))
-    numTable[int(splitLine[0])] = (splitLine[1],float(splitLine[6]),int(splitLine[7]),\
-                                   int(splitLine[5]),float(splitLine[2]),float(splitLine[3]),
+    numTable[int(splitLine[0])] = (splitLine[1], float(splitLine[6]), int(splitLine[7]),
```

Comment:
It would be nice if these code formatting changes were not mixed in with the substantive changes. I know it happens sometimes, but it's easy to `git commit` just the lines you've changed and do the formatting changes as a separate PR.
I am curious why these are still coming though; we already had a big block that, I thought, did everything?